### PR TITLE
Updated the README and added gitattributes and gitignore file

### PR DESCRIPTION
**Pull request by freemansw1**
Hi! I updated the README and added .gitattributes and .gitignore files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+*.grb2
+
+bin.dp.grib1/example_data/*
+bin.dp.grib2/example_data/*
+dp-p[0-9]*
+bin.revu/example_data/*
+*.swp
+*.docx
+misc/hdf5-1.8.9*
+misc/mpich2-1.4.1.precomp.binaries/*
```
Comment (ldgrantcsu) [inline .gitignore]: Is this repetitive of misc/mpich2-* ?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+*.h5
```
Comment (ldgrantcsu) [inline .gitignore:2]: Let's add some comments!

**Review by ldgrantcsu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (freemansw1) [reply .gitignore]: Yes! I have resolved this in my new version.

File: .gitignore (hunk 2 above)
Comment (freemansw1) [reply .gitignore:2]: I've added  comments throughout this file.

**Review by ldgrantcsu — APPROVED** [review]

**Pull request merged**